keras/utils/training_utils.py

```diff
@@ -95,14 +95,24 @@ def multi_gpu_model(model, gpus):
     if K.backend() != 'tensorflow':
         raise ValueError('`multi_gpu_model` is only available '
                          'with the TensorFlow backend.')
-    if gpus <= 1:
-        raise ValueError('For multi-gpu usage to be effective, '
-                         'call `multi_gpu_model` with `gpus >= 2`. '
-                         'Received: `gpus=%d`' % gpus)
+    if isinstance(gpus, int):
```

Comment:
Do the reverse: `if isinstance(gpus, (list, tuple)):`/`else`; no need to typecheck `int`